File: entrega2/menu/src/main/java/com/aps/menu/entidades/Endereco.java
package com.aps.menu.entidades;

import javax.persistence.Entity;

@Entity
public class Endereco {
    private String rua, bairro;
    private int cep, numero;

    public Endereco(String rua, String bairro, int cep, int numero) {
        this.rua = rua;
        this.bairro = bairro;
        this.cep = cep;
        this.numero = numero;
    }

    public Endereco() {
        
    }
}

File: soa/menu-app/acesso-service/src/entidades/endereco.ts
export class Endereco {
    private rua: string;
    private cep: number;
    private bairro: string;
    private numero: number;

    constructor(rua: string, cep: number, bairro: string, numero: number) {
        this.rua = rua;
        this.cep = cep;
        this.bairro = bairro;
        this.numero = numero;
    }

    public getRua() : string {
        return this.rua;
    }
    
    public getCep() : number {
        return this.cep;
    }

    public getBairro() : string {
        return this.bairro;
    }

    public getNumero() : number {
        return this.numero;
    }
}